_posts/2020-06-20-snap-minis.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Snap Minis: How we can help you bring your app into SnapChat"
+date: 2020-06-20 11:00:00
+reviewed: 2020-06-20 11:00:00
+categories: []
+author: ABizzinotto
+---
+
+Last week, [Snap](https://www.snap.com) announced a new product during the [Snap Partner Summit 2020](https://www.youtube.com/watch?v=BfvubHha69k), [Snap Minis](https://minis.snapchat.com/). Minis are bite-sized third-party applications that live inside [Snapchat](https://www.snapchat.com/) and can be accessed through Snapchat's Chat interface. They are built on the same principle and technology as the Snap Games and are easy to develop, allowing you to have an app that takes advantage of the existing [Snapchat](https://www.snapchat.com/) application and is available to Snapchat users worldwide.
+
+OmbuLabs recently developed a Snap Mini application for one of our clients. In this article I'll share with you the main advantages of launching a Snap Mini and how we can help you get in on this.
+
+<!--more-->
+
+First of all, why create a Snap Mini? Here are the main advantages of having your own app inside [Snapchat](https://www.snapchat.com/):
```

Comment:
We can also add a title like "Painless and controlled distribution/deployment" (or something else) saying that snapchat provides a CMS to manage builds easily and publish them either for testing to a limited set of users or for production worldwide or for specific countries.

Reply:
We cannot disclose that information right now. Later, when we write a case study with Snap's approval, yes. Right now, we can't go too far beyond what Snap has disclosed.